Source/Plugin.BLE/Shared/Contracts/IDevice.cs

```diff
@@ -106,5 +106,18 @@ public interface IDevice : IDisposable
         /// <returns>True if the update request was sucessfull. On iOS it will always return false.</returns>
         /// <param name="interval">The requested interval (High/Low/Normal)</param>
         bool UpdateConnectionInterval(ConnectionInterval interval);
+
+
+        /// <summary>
+        /// Gets the information if the device has hinted during advertising that the device is connectable.
+        /// This information is not pat of an advertising record. It's determined from the PDU header.
+        /// Check CanIsConnectable to verify that the device supports IsConnectable.
```

Comment:
There is another reference to 'CanIsConnectable' here that should be changed to 'SupportsIsConnectable'.